README.md

```diff
@@ -1,6 +1,6 @@
 # Apache Spark
 
-Spark is a unified analytics engine for large-scale data processing. It provides
+Apache Spark is a unified analytics engine for large-scale data processing. It provides
```

Comment:
(Is this intentional? it's already called Apache Spark just above so it'd be fine as is)

Reply:
lol i'm triggering tests. I should revert ;)

Reply:
done!